src/components/editor/EditorArticleListController.jsx

```diff
@@ -2,20 +2,21 @@ import React from 'react';
 import { browserHistory } from 'react-router';
 import FalcorController from "lib/falcor/FalcorController";
 import List from "./List";
+import _ from "lodash";
 
-var pageLength = 50;
+const NUM_ARTICLES_IN_PAGE = 50;
 
-export default class ArticleListController extends FalcorController {
+export default class EditorArticleListController extends FalcorController {
   constructor(props) {
     super(props);
     this.handleSubmit = this.handleSubmit.bind(this);
     this.handleUpdateSlugValue = this.handleUpdateSlugValue.bind(this);
-    this.safeSetState({
+    this.state = {
       slugSearchValue: ""
-    })
+    };
   }
   static getFalcorPath(params) {
-    return [['articlesByTime', 'paginated', {from: pageLength*(params.page-1), length: pageLength}, ['title', 'slug']], ['articlesByTime', 'paginated', 'total']];
+    return [['articlesByTime', 'paginated', {from: NUM_ARTICLES_IN_PAGE*(params.page-1), length: NUM_ARTICLES_IN_PAGE}, ['title', 'slug']], ['articlesByTime', 'paginated', 'total']];
```

Comment:
Have you already changed articlesBytime as we discussed or are still going to do it?